packages/core/src/components/Tabs/index.tsx

```diff
@@ -171,10 +176,18 @@ export default withBoundary('Tabs')(
       display: 'flex',
     },
 
-    nav_borderless: {
+    nav_noBorder: {
       borderColor: color.clear,
     },
 
+    nav_secondary: {
+      padding: unit / 2,
+      overflow: 'auto',
```

Comment:
this might conflict with the gradient scroller? add a story of secondary & scrollable enabled

Reply:
can you add a screenshot confirming this doesn't conflict with scrollable? otherwise, i can pull down this branch and check it out locally in a bit.

Reply:
![demo](https://user-images.githubusercontent.com/205004/66510293-9a338e80-ea89-11e9-9603-035b724186a4.gif)